frontend/constants/paths.ts

```diff
@@ -29,6 +29,10 @@ export const PATHS = {
     path: '/profile',
     title: 'プロフィール',
   },
+  FAVORITE_VIDEO: {
+    path: '/videos/favorite',
+    title: 'プロフィール',
+  },
```

Comment:
```suggestion
  VIDEO: {
  FAVORITE: {
    path: '/videos/favorite',
    title: 'プロフィール',
    }
  },
```

みたいにvideoが入ってきた時も考えて追加してもらえると！

Reply:
なるほど、勉強になります！